version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.9.9",
+  "version": "2.9.10",
```

Comment:
Can you please integrate `2.9.10` in project-apollo repo once it is released, similar to https://github.com/Azure/project-apollo/pull/1117.